docs/azure/GettingStarted/FirmwareConfiguration.rst

```diff
@@ -5,7 +5,17 @@ Configure the firmware
 
 To configure the :ref:`firmware <firmware-azure-index>`, complete the following mandatory steps:
 
-1. // FIXME: Implement
+1. Run the following command to print the Azure IoT DPS ID scope to which your devices use during device provisioning:
```

Comment:
Is it 'register'? Or is it :  Run the following command to print the Azure IoT DPS ID scope, which your devices use during device provisioning:
```suggestion
1. Run the following command to print the Azure IoT DPS ID scope to which your devices register during device provisioning:
```

Reply:
No, the ID scope is used to identify the DSP instance. It is the same for all devices.